test/test_public_bindings.py

```diff
@@ -133,6 +133,7 @@ def test_no_new_bindings(self):
             "GraphExecutorState",
             "has_cuda",
             "has_cudnn",
+            "has_kleidiai",
```

Comment:
This should be removed now that it is expected to be read from torch.backends?

Reply:
Hello AlbanD, can you please check this https://github.com/pytorch/pytorch/pull/134124#discussion_r1757305689
and https://github.com/pytorch/pytorch/actions/runs/12355469852/job/34482254051?pr=134124

We have removed "has_kleidiai" from test_public_bindings and the ci is failing  https://github.com/pytorch/pytorch/actions/runs/12355469852/job/34482254051?pr=134124

Reply:
How can we make this change and pass the CI?

Reply:
Update: I have again reverted this change unless we get a solution for the CI failure.

Reply:
Got it, changes are done and pushed